hack/update-codegen.sh

```diff
@@ -33,5 +33,5 @@ bash "${CODEGEN_PKG}"/generate-groups.sh \
   all \
   sigs.k8s.io/scheduler-plugins/pkg/generated \
   sigs.k8s.io/scheduler-plugins/pkg/apis \
-  podgroup:v1alpha1 \
-  --go-header-file "${SCRIPT_ROOT}"/hack/boilerplate/boilerplate.generatego.txt
+  "scheduling:v1alpha1" \
```

Comment:
Should be "podgroup:v1alpha1 elasticquota:v1alpha1".

Reply:
Take back my words, as it's fine to put ElasticQuota and PodGroup into the same API group, while also letting them evolve with different versions.